.github/workflows/draw-zmk.yml

```
@@ -97,9 +102,11 @@ jobs:
               draw_args=$(get_args "${{ inputs.draw_args }}" "$keyboard")
               echo "INFO:   got extra draw args: $draw_args"
 
-              if [ -f "config/${keyboard}.json" ]; then
-                  echo "INFO:   found config/${keyboard}.json";
-                  draw_args+=" -j config/${keyboard}.json"
+              json_path="${{ inputs.json_path }}"
+              [ -e "$json_path" ] && json_dir="$json_path"
+              if [ -f "${json_dir:-config}/${keyboard}.json" ]; then
+                  echo "INFO:   found ${json_dir:-config}/${keyboard}.json";
+                  draw_args+=" -j ${json_dir:-config}/${keyboard}.json"
               fi
```

```suggestion
              json_path="${{ inputs.json_path }}"
              if [ -f "$json_path/${keyboard}.json" ]; then
                  echo "INFO:   found $json_path/${keyboard}.json";
                  draw_args+=" -j $json_path/${keyboard}.json"
              fi
```
(not tested, would appreciate if you could)
I think it is safe to simplify this by not falling back to `config` if the `json_path` argument is explicitly specified by the user. (If not specified it defaults to `config` anyway.)